pbft/HyperMake

```diff
@@ -44,18 +36,6 @@ targets:
     artifacts:
       - bin/linux/truechain-engine
 
-  build-darwin:
```

Comment:
so you've updated Dockerfile with xgo but removed build-darwin?

Reply:
No. If you check the msg in gitter, this is the quick fix.

Reply:
but then why are we importing xgo in Dockerfile in the same commit?